scripts/setup/setup_acm_managed_mce.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# Run this script on the ACM hub cluster to import managed MCE clusters into ACM
+# as well as auto-importing the HCPs on those managed MCE clusters.
+
+## Requires:
+# kubeconfigs for the managed MCE clusters to bring into ACM in the kubeconfigs/ directory
+## ensure kubeconfigs are named as <cluster-name>.kubeconfig
+# envsubst
+# oc or kubectl
+# clusteradm
+
+MANIFESTS_DIR="resources/managedMCE"
+MCE_CONFIGS_DIR="resources/managedMCE/kubeconfigs"
+
+export ADDON_CONFIG_NS=${ADDON_CONFIG_NS:-"addon-ns-config"}
+export AGENT_NS=${AGENT_NS:-"open-cluster-management-agent-addon-discovery"}
+export KLUSTERLET_CONFIG=${KLUSTERLET_CONFIG:-"mce-import-klusterlet-config"}
+export MCE_NS=${MCE_NS:-"multicluster-engine"}
+export TIMEOUT=${TIMEOUT:-"300s"}
+
+envsubst < $MANIFESTS_DIR/00-addondeploymentconfig.yaml | oc apply -f -
+
+# Update ClusterManagementAddOn for work-manager addon to add a reference to the AddOnDeploymentConfig resource created in the previous step.
+# Do the same update for managed-serviceaccount addon.
+envsubst < $MANIFESTS_DIR/01-clustermanagementaddon-work-mgr.yaml | oc apply -f -
+envsubst < $MANIFESTS_DIR/02-clustermanagementaddon-sa.yaml | oc apply -f -
```

Comment:
can you do the same clustermanagementaddon configuration for `cluster-proxy` addon as well?